docs/installation/aks.md

```diff
@@ -0,0 +1,212 @@
+- `CLUSTER_NAME`: The name of the AKS cluster. The default value is `kubeai-stack`.
+- `USER_NAME`: The username for the AKS cluster. The default value is `azureuser`.
+- `GPU_NODE_POOL_NAME`: The name of the GPU node pool. The default value is `gpunodes`.
+- `GPU_NODE_COUNT`: The number of GPU nodes in the GPU node pool. The default value is `1`.
+- `GPU_VM_SIZE`: The SKU of the GPU VMs in the GPU node pool. The default value is `Standard_NC48ads_A100_v4`. You can find more GPU VM sizes [here](https://learn.microsoft.com/en-us/azure/virtual-machines/sizes/overview#gpu-accelerated).
+
+## 2. Create a Resource Group
+
+```bash
+az group create \
+  --name "${AZURE_RESOURCE_GROUP}" \
+  --location "${AZURE_REGION}"
+```
+
+## 3. Create an AKS Cluster (CPU Only)
+
+If you only intend to run CPU-based models, you can create a simple AKS cluster as follows:
+
+```bash
+az aks create \
+    --resource-group "${AZURE_RESOURCE_GROUP}" \
+    --name "${CLUSTER_NAME}" \
+    --enable-oidc-issuer \
+    --enable-workload-identity \
+    --enable-managed-identity \
+    --node-count 1 \
+    --location "${AZURE_REGION}" \
+    --admin-username "${USER_NAME}" \
+    --generate-ssh-keys \
+    --os-sku Ubuntu
+```
+
+## 4. (Optional) Add a GPU Node Pool
+
+If you want to run GPU-backed models, you need a GPU node pool. Below is an example using the `Standard_NC48ads_A100_v4` SKU. Feel free to adjust `--node-vm-size` to match a GPU SKU you have quota for:
+
+```bash
+az aks nodepool add \
+  --resource-group "${AZURE_RESOURCE_GROUP}" \
+  --cluster-name "${CLUSTER_NAME}" \
+  --name "${GPU_NODE_POOL_NAME}" \
+  --node-count "${GPU_NODE_COUNT}" \
+  --node-vm-size "${GPU_VM_SIZE}" \
+  --enable-cluster-autoscaler \
+  --min-count 1 \
+  --max-count 3
+```
+
+## 5. Get AKS Credentials
+
+Download and merge the kubeconfig for your AKS cluster:
+
+```bash
+az aks get-credentials \
+  --resource-group "${AZURE_RESOURCE_GROUP}" \
+  --name "${CLUSTER_NAME}" \
+  --overwrite-existing
+```
+
+## 6. Install the NVIDIA Device Plugin (for GPU Clusters)
+
+If you created a GPU node pool, install the NVIDIA device plugin so Pods can request GPUs:
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/NVIDIA/k8s-device-plugin/v0.17.1/deployments/static/nvidia-device-plugin.yml
+```
+
+Confirm the DaemonSet is running:
+
+```bash
+kubectl get daemonset nvidia-device-plugin-daemonset -n kube-system
+```
+
+## 7. Install KubeAI
+
+Add the KubeAI Helm repository and update:
+
+```bash
+helm repo add kubeai https://www.kubeai.org
+helm repo update
+```
+
+**(Optional)** If you plan to use private or TOS-protected models from Hugging Face, export your token:
+
+```bash
+export HUGGING_FACE_HUB_TOKEN=<YOUR_HF_TOKEN>
+```
+
+Install KubeAI:
+
+```bash
+helm upgrade --install kubeai kubeai/kubeai \
+  --set secrets.huggingface.token="${HUGGING_FACE_HUB_TOKEN}" \
+  --wait
+```
+
+### Customizing Resource Profiles
```

Comment:
we should remove this section about Customizing Resource Profiles and instead point to the k8s device plugin values file, which has the resourceProfiles for NVIDIA device plugin.

AKS is using K8s device plugin, so we can re-use the values file mentions in this guide: https://www.kubeai.org/installation/any/#installation-using-nvidia-gpus

Link to values file for nvidia device plugin: https://github.com/substratusai/kubeai/blob/main/charts/kubeai/values-nvidia-k8s-device-plugin.yaml

Reply:
Okay sure, all done!